{"uid":"45349234d7bead6d","name":"User sends a correct data into the form","fullName":"ui_tests.test_form_from_main_page.TestMainPageForm#test_positive_form_data","historyId":"350f59d9efd3056da7a4a44ebe3ee198","time":{"start":1670850201561,"stop":1670850210471,"duration":8910},"description":"User is scrolling to the bottom and sends correct data","descriptionHtml":"<p>User is scrolling to the bottom and sends correct data</p>\n","status":"failed","statusMessage":"AssertionError: Not success popup presented","statusTrace":"self = <ui_tests.test_form_from_main_page.TestMainPageForm object at 0x7fb5a30a6fd0>\nbrowser = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"1aadf459-a94f-48ae-a015-a6632f28be73\")>\npositive_data_case = [(('name', 'name'), 'test_user'), (('name', 'email'), 'test@fojin.tech'), (('name', 'phone'), '12345678901'), (('name', 'about'), 'some useless information')]\n\n    @allure.title('User sends a correct data into the form')\n    @severity(severity_level.CRITICAL)\n    @allure.feature('User sends correct data')\n    @allure.description('User is scrolling to the bottom and sends correct data')\n    def test_positive_form_data(self, browser, positive_data_case: list):\n        \"\"\"\n          Test fills the application with correct data and checks the popup answer\n        \"\"\"\n        page = MainPage(browser, self.url)\n        page.open()\n        for data in positive_data_case:\n            page.input_data_to_form(data)\n        page.scroll_to_and_click_element(FormLocators.SEND_BUTTON)\n        page.expl_wait_for_elem_visibility(FormLocators.POPUP)\n>       page.check_popup_is_presented(FormLocators.POPUP)\n\nui_tests/test_form_from_main_page.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ui_tests.pages.main_page.MainPage object at 0x7fb5a2f72e50>\nlocator = ('css selector', '#form > div > div > div')\n\n    def check_popup_is_presented(self, locator: tuple):\n        \"\"\"\n        Test checks that success message appears\n        \"\"\"\n        popup = self.browser.find_element(*locator)\n        print(popup.text)\n>       assert popup.is_displayed(), 'Not success popup presented'\nE       AssertionError: Not success popup presented\n\nui_tests/pages/main_page.py:80: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"test_failed_check","time":{"start":1670850196864,"stop":1670850196864,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"browser","time":{"start":1670850196864,"stop":1670850201561,"duration":4697},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"positive_data_case","time":{"start":1670850201561,"stop":1670850201561,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"User is scrolling to the bottom and sends correct data","status":"failed","statusMessage":"AssertionError: Not success popup presented","statusTrace":"self = <ui_tests.test_form_from_main_page.TestMainPageForm object at 0x7fb5a30a6fd0>\nbrowser = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"1aadf459-a94f-48ae-a015-a6632f28be73\")>\npositive_data_case = [(('name', 'name'), 'test_user'), (('name', 'email'), 'test@fojin.tech'), (('name', 'phone'), '12345678901'), (('name', 'about'), 'some useless information')]\n\n    @allure.title('User sends a correct data into the form')\n    @severity(severity_level.CRITICAL)\n    @allure.feature('User sends correct data')\n    @allure.description('User is scrolling to the bottom and sends correct data')\n    def test_positive_form_data(self, browser, positive_data_case: list):\n        \"\"\"\n          Test fills the application with correct data and checks the popup answer\n        \"\"\"\n        page = MainPage(browser, self.url)\n        page.open()\n        for data in positive_data_case:\n            page.input_data_to_form(data)\n        page.scroll_to_and_click_element(FormLocators.SEND_BUTTON)\n        page.expl_wait_for_elem_visibility(FormLocators.POPUP)\n>       page.check_popup_is_presented(FormLocators.POPUP)\n\nui_tests/test_form_from_main_page.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ui_tests.pages.main_page.MainPage object at 0x7fb5a2f72e50>\nlocator = ('css selector', '#form > div > div > div')\n\n    def check_popup_is_presented(self, locator: tuple):\n        \"\"\"\n        Test checks that success message appears\n        \"\"\"\n        popup = self.browser.find_element(*locator)\n        print(popup.text)\n>       assert popup.is_displayed(), 'Not success popup presented'\nE       AssertionError: Not success popup presented\n\nui_tests/pages/main_page.py:80: AssertionError","steps":[],"attachments":[{"uid":"1840a08dd09ad55a","name":"log","source":"1840a08dd09ad55a.txt","type":"text/plain","size":1118},{"uid":"e107dc7ace925bd3","name":"stdout","source":"e107dc7ace925bd3.txt","type":"text/plain","size":1},{"uid":"57c3205f36112841","name":"stderr","source":"57c3205f36112841.txt","type":"text/plain","size":501}],"parameters":[],"stepsCount":0,"attachmentsCount":3,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"browser::0","time":{"start":1670850210507,"stop":1670850211220,"duration":713},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"test_failed_check::0","time":{"start":1670850211221,"stop":1670850211227,"duration":6},"status":"broken","statusMessage":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=42261): Max retries exceeded with url: /session/1aadf459-a94f-48ae-a015-a6632f28be73/screenshot (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb5a2fcfd50>: Failed to establish a new connection: [Errno 111] Connection refused'))\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 916, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/fojin_site/fojin_site/conftest.py\", line 61, in test_failed_check\n    screenshot(driver, request.node.nodeid)\n  File \"/home/runner/work/fojin_site/fojin_site/conftest.py\", line 66, in screenshot\n    allure.attach(browser.get_screenshot_as_png(), name=f\"Screenshot fail_{name}\",\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 965, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 977, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 442, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 294, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 316, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/request.py\", line 74, in request\n    return self.request_encode_url(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/request.py\", line 96, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 815, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 815, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 815, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 787, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/urllib3/util/retry.py\", line 592, in increment\n    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"labels":[{"name":"feature","value":"User sends correct data"},{"name":"severity","value":"critical"},{"name":"parentSuite","value":"ui_tests"},{"name":"suite","value":"test_form_from_main_page"},{"name":"subSuite","value":"TestMainPageForm"},{"name":"host","value":"fv-az443-57"},{"name":"thread","value":"3090-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"ui_tests.test_form_from_main_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser","value":"'firefox'"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"45349234d7bead6d.json","parameterValues":["'firefox'"]}